prow/spyglass/viewers/metadata/metadata_viewer.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package metadata provides a metadata viewer for Spyglass
+package metadata
+
+import (
+	"bytes"
+	"encoding/json"
+	"time"
+
+	"html/template"
+
+	"github.com/sirupsen/logrus"
+	"k8s.io/test-infra/prow/spyglass/viewers"
+)
+
+const (
+	name  = "MetadataViewer"
+	title = "Metadata"
+)
+
+func init() {
+	viewers.RegisterViewer(name, title, ViewHandler)
+}
+
+// Started is used to mirror the started.json artifact
+type Started struct {
```

Comment:
We should declare these in some more common place -- there is already some version of this in pod-utils as well

Reply:
Definitely open to using other package's definitions, this viewer is essentially a stub at the moment.

Reply:
@stevekuznetsov im beginning work on this viewer, could you point me to the version on pod-utils i should be using?